lib/miq_automation_engine/engine/miq_ae_service.rb

```diff
@@ -58,9 +58,12 @@ def initialize(ws, inputs = {}, body = nil, logger = $miq_ae_logger)
       self.body              = body if body
       @persist_state_hash    = ws.persist_state_hash
       @logger                = logger
+      ws.disable_rbac
```

Comment:
@kbrock 
https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/engine/drb_remote_invoker.rb#L14
Each service has a workspace
Each front also has a workspace
https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/engine/drb_remote_invoker.rb#L35

We use the MiqAeService.find based on the service objects object_id
https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/engine/drb_remote_invoker.rb#L15